en/vespa-quick-start-cli.html

```diff
@@ -39,7 +39,7 @@ <h2 id="running-vespa-in-docker">Running Vespa in Docker</h2>
 <li>
 <p><strong>Install the Vespa CLI:</strong></p>
 <pre data-test="exec">
```

Comment:
This probably shouldn't be executed by the documentation tests, as that environment doesn't have `brew`.

Reply:
good point, will remove - the binary is installed in the image used afaik